static_files/css/style.css

```diff
@@ -36,4 +57,43 @@ main{
     line-height: 22px;
 }
 
+.CathoLogo {
+
+    align-self: center;
+}
+
+
+
+.VisibleMenu {
+
+    color: #CCCCCC;
+    font-weight: bold;
+}
```

Comment:
What do you think about defining a centralized alignment of the contents of this class? Comparing the prototype with the implementation, it is possible to see that in the first case it is apparently aligned with the logo.
Implementation
![image](https://user-images.githubusercontent.com/105747728/234890670-be4bef67-c7dd-4d40-9e7f-869dc31da6d7.png)
Prototype
![image](https://user-images.githubusercontent.com/105747728/234890738-9384ab72-00aa-4c9c-bcab-97abfe3819c7.png)

Reply:
Thanks for pointing it out! Fixed!